src/var.cc

```diff
@@ -161,7 +162,7 @@ void RecursiveVar::AppendVar(Evaluator* ev, Value* v) {
 }
 
 void RecursiveVar::Used(Evaluator* ev, const Symbol& sym) const {
-  if (SelfReferential()) {
+  if (ev->SelfReferential(this)) {
```

Comment:
Remind me again, what causes the recursive evaluation to stop?

Reply:
ERROR_LOC will exit the program with a code of 1.